pkg/cobrautl/help.go

```diff
@@ -151,6 +151,7 @@ func getSubCommands(cmd *cobra.Command) []*cobra.Command {
 	return subCommands
 }
 
+// Deprecated: Please use go.etcd.io/etcd/etcdctl/v3/util instead.
```

Comment:
The linter error is complaining that the comment isn't starting with `UsageFunc`
```suggestion
// UsageFunc is the usage function for the cobra command.
// Deprecated: Please use the UsageFunc in go.etcd.io/etcd/etcdctl/v3/util instead.
```

Reply:
Oh, I see thanks.